src/main/java/seedu/address/model/person/Archive.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a person's archive status
+ */
+public class Archive {
+    public static final String MESSAGE_CONSTRAINTS = "Archive should be boolean";
+
+    public final String value;
+
+    /**
+     * Constructs a {@code Archive}.
+     *
+     * @param isArchived A boolean.
+     */
+    public Archive(boolean isArchived) {
+        value = Boolean.toString(isArchived);
+    }
+
+    /**
+     * Constructs a {@code Archive} from Json data
+     */
+    public Archive(String archive) {
+        requireNonNull(archive);
+        checkArgument(isValidArchive(archive), MESSAGE_CONSTRAINTS);
+        this.value = archive;
+    }
```

Comment:
I doubt it is necessary but I was thinking that it might be something nice to have.
As I was thinking that in the code itself, it would be better to have a constructor to take in `boolean` since it is easier to input `boolean` as an argument instead of `String`, and it also ensures less room for human-error if the constructor with `boolean isArchived` is used, since there is only 2 argument, `true` or `false` that users can input. 
But at the same time the alternative constructor which takes in a `String archive` is needed to read the json file - which is why I decided to have 2 constructors for the `Archive` command.

Reply:
Thanks for clarifying! I now understand why you would want to have 2 constructors instead of 1! 👍